extension/persistence/relational-jdbc/src/main/java/org/apache/polaris/extension/persistence/relational/jdbc/JdbcBasePersistenceImpl.java

```diff
@@ -428,7 +443,7 @@ public <T> Page<T> listEntities(
       List<T> resultsOrEmpty =
           results == null
```

Comment:
IJ says `results == null` is always false :thinking:

Reply:
True!